app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -76,6 +77,8 @@ class DefaultBrowserToolbarController(
     private val currentSession
         get() = customTabSession ?: activity.components.core.sessionManager.selectedSession
 
+    private val navOptions by lazy { NavOptions.Builder() }
```

Comment:
Very small nit, but since this is only getting used in `getToolbarNavOptions` maybe it's reasonable to move it to being a local variable? Or do you think we'll need it outside of that function?